guides/v2.1/ui_comp_guide/concepts/ui_comp_linking_concept.md

```diff
@@ -41,7 +42,7 @@ Example of setting `exports` in a component's `.js` file:
 }
 {% endhighlight js%}
 
-Here `visible` is the `key`, `${ $.provider }:visibility` is the `value`. The value of the local `visible` property is assigned to the `visibility` property of the `provider` component. The latter is changed automatically if the value of `visible` changes.
+Here `visible` is the `key`, `${ $.provider }:visibility` is the `value`. The value of the local `visible` property is assigned to the `visibility` property of the `provider` component. The latter is changed automatically if the value of `visible` changes if the local `visible` property is observable (which it isn't given only the code example above).
```

Comment:
>if the local `visible` property is observable (which it isn't given only the code example above)

This is not correct, you can change value of `visible` with `set` method and it will trigger export.

Reply:
It is correct based on the code sample. If one of the parent classes declares the property as observable (using default.tracks or a manual call) then it works, but that is not included in the displayed code example.